src/main/java/gov/cabinetofice/gapuserservice/service/SpotlightService.java

```diff
@@ -43,6 +44,9 @@ public class SpotlightService {
     private String codeVerifier;
     private String codeChallenge;
 
+    private final String ACCESS_TOKEN_NAME = "access_token";
```

Comment:
these can be static:

`    private static final String ACCESS_TOKEN_NAME = "access_token";
     private static final String REFRESH_TOKEN_NAME = "refresh_token";
`

Reply:
Fixed